README.md

```diff
@@ -95,6 +95,27 @@ which will significantly speed up the incremental build process.
 
 After running `mvn package` for the first time, you should run ``k-distribution/target/release/k/bin/k-configure-opam; eval `opam config env` ``. This performs first-time setup of the OCAML backend.
 
+## Installing on fresh Windows Subsystem for Linux
+
+1. Install the Ubuntu package from the Windows Store, which as of now is an alias for the Ubuntu LTS 18.04 package. 
+2. Download the latest Ubuntu Bionic distribution from https://github.com/kframework/k/releases 
+    to a temporary directory, for example `d:\temp`
+    
+3. Optional. By default the WSL bash user is root. It is recommended to run K as a non-root user, otherwise K will print warnings for each executed command.
+Follow for example [this guide](https://www.tenforums.com/tutorials/128052-add-user-windows-subsystem-linux-wsl-distro-windows-10-a.html) to create a new user and [this guide](https://askubuntu.com/questions/816732/how-to-change-default-user-in-wsl-ubuntu-bash-on-windows-10) to make it the default user for `bash`.  
+
+4. Open linux bash. 
+5. Run the following commands:
+    $ `sudo apt-get update`
+    $ `cd <download dir>`. In our example it is `cd /mnt/d/temp`
+    $ `sudo apt-get install ./kframework_5.0.0_amd64_bionic.deb`
+        This will install ~1.4GB of dependencies and will take some time.
+        F will be installed to `/usr/lib/kframework`
+6. Now you can try to run some programs:
+    $ `cd /usr/lib/kframework/tutorial/2_languages/1_simple/1_untyped`
```

Comment:
this will not work as this directory will be owned by root. You need to tell people to `cp -R /usr/lib/kframework/tutorial ~` and then tell them to build the version of the tutorial they copied into their home directory.